src/ext/background.js

```diff
@@ -690,3 +694,13 @@ function setConfig(val) {
         countStoredTokens();
     }
 }
+
+/**
+ * Checks that the URL initiating the config switch isn't specifically
+ * ignored
+ * @param {string} url URL string to check
+ * @return {boolean} true if ignored
+ */
+function ignoredUrl(url) {
+    return switchConfigIgnoreUrls().some((u) => url.includes(u));
```

Comment:
This would also match `http://not.hcaptcha.com/getcaptcha`. or `https://example.com/not/hcaptcha.com/getcaptcha-fake`. Is that a problem?